### ISSUE TITLE
Pause button switches to "resting"

### ISSUE THREAD

**Issue by hsch** [mfbieber/PomodoroTimer#5]
I feel like the pause button should just pause whatever phase the timer is currently in. "Play" should then resume the countdown (no reset in that case either).

**Comment by mfbieber** [comment]
This is actually on purpose. Pause = Rest. ;) Play = Work.

**Comment by hsch** [comment]
Yeah, that's what I thought. Still, it was surprising for me as a user, and it also didn't make too much sense in my opinion: Pause is pause. ¯\_(ツ)_/¯ Also, there is already the "reset" button which resets to rest.

**Comment by mfbieber** [comment]
No, reset resets everything! 

Have a look at these icons: https://github.com/Fitbit/sdk-design-assets/tree/master/icons

Do you think something else is better?

**Comment by hsch** [comment]
It's fine, resetting resets the cycle, which starts with resting.

**Comment by mfbieber** [comment]
I mean, also for the other buttons ;)

**Comment by mfbieber** [comment]
So pausing means you are taking a break from work. Maybe a coffee cup or something is better? Or maybe I can add the possibility to configure the icons yourself via the settings? ;)

**Comment by hsch** [comment]
I would not make the buttons configurable. Better to have a well opinionated app that "just works".

tbh, I still don't understand the value of a "rest" button (ignoring for a moment that the icon is confusing), why I shouldn't use reset, and why I wouldn't want to pause.

Let's say I'm in the middle of a 25 minute work period, but need to pause, I really do want to pause for however long I need it (e.g. for a phone call), not reset (or switch back to resting).

If pausing is not part of the Pomodoro concept, remove that button altogether?

Anyway; I'm just providing feedback from my perspective as an independent user, and I find the current behavior too surprising. :(

**Comment by mfbieber** [comment]
Yes, thanks for your feedback. I appreciate it! I will make your suggestions work, so differentiate between pause and rest and provide different buttons for that. Maybe the user can also just pick what kind of buttons he wants to have and choose if he wants a rest, a pause, a reset, or whatever button :)
